test/java/mjc/ProductTest.java

```diff
@@ -0,0 +1,16 @@
+package test.java.mjc;
+
+import main.java.mjc.Product;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class ProductTest {
+
+    @Test
+    void testToString() {

```

Comment:
i consider this test case as for the constructor but not the method. 
because the constructor is trivial then this test case is excessive. 
someone may argue.